src/routes/ProtectedRoute.jsx

```diff
@@ -0,0 +1,19 @@
+import { Navigate, Outlet } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const ProtectedRoute = ({
+    isAllowed,
```

Comment:
Set default values for undefined fields:
```
isAllowed = false,
component = null,
```
It works without it (since `undefined` is falsish), but this way you set values with necessary types. Moreover, prop types are defined below.

Reply:
has been fixed